woocommerce-payments.php

```diff
@@ -8,9 +8,9 @@
  * Woo: 5278104:bf3cf30871604e15eec560c962593c1f
  * Text Domain: woocommerce-payments
  * Domain Path: /languages
- * WC requires at least: 7.8
- * WC tested up to: 8.0.0
- * Requires at least: 6.1
+ * WC requires at least: 7.6
+ * WC tested up to: 7.8.0
```

Comment:
I'm tempted to leave this as 6.3 but goal of this PR is to revert https://github.com/Automattic/woocommerce-payments/pull/7050, so I went ahead to revert this line as well.